### DISCUSSION TITLE
Setting up anonymous authentication for embedded content

### DISCUSSION THREAD

**Discussion by lullu57** [telekom-security/tpotce#1198] [Q&A] [answered]
Hi, 

I'm trying to setup anonymous authentication for Kibana to be able to show an embed version of the T-Pot live attack map on our website as described here:
https://www.elastic.co/guide/en/kibana/current/kibana-authentication.html#anonymous-authentication 
https://www.elastic.co/guide/en/kibana/master/kibana-authentication.html#embedded-content-authentication

My issue has been to try to create the described user necessary for the authentication through Kibana on T-Pot as the Management / Security / Users page is not available on T-Pots Kibana. I understand that this is a Kibana issue and not a specifically T-pot issue, however, would it be possible to guide me as to why this section is not available, and to what are the options to be able to embed such T-pot specific dashboards/Kibana objects? 

Appreciate any help :)

**Comment by t3chn0m4g3** [comment, marked as answer]
For T-Pot we do not support this out of the box as we decided to keep things as simple as possible. You can however check out the Wiki to learn about volumes and adjusting the configs to forward logs to your own ELK instance.

**Reply by lullu57** [reply]
I had tried to do this initially however did not manage to set up the connection. Do you by any chance have the appropriate links that go through how to set it up maybe I can give it another try?
